client/ayon_core/pipeline/colorspace.py

```diff
@@ -755,30 +782,44 @@ def _get_global_config_data(
 
     profile_type = profile["type"]
     if profile_type in ("builtin_path", "custom_path"):
-        template = profile[profile_type]
-        result = StringTemplate.format_strict_template(
-            template, template_data
-        )
-        normalized_path = str(result.normalized())
-        if not os.path.exists(normalized_path):
-            log.warning(f"Path was not found '{normalized_path}'.")
-            return None
-
-        return {
-            "path": normalized_path,
-            "template": template
-        }
+        return _get_config_path_from_profile_data(
+            profile, profile_type, template_data)
 
     # TODO decide if this is the right name for representation
     repre_name = "ocioconfig"
 
+    # Backward compatibility for old projects
+    # TODO remove in future 0.4.5 onwards
+    product_name = profile.get("product_name")
```

Comment:
Why are used `.get(` methods here? Settings for this version of ayon-core will always contain the values if they are in model.

Reply:
yeah, you are right.

Reply:
here is the improvement f9e0b05bf2dac3f596a9c53f32368219072970dd